resource/ui/ts/models/log.ts

```diff
@@ -0,0 +1,149 @@
+/// <reference path="../util/format.ts" />
+/// <reference path="../util/querycache.ts" />
+// Author: Bram Gruneir (bram+code@cockroachlabs.com)
+
+/**
+ * Models contains data models pulled from cockroach.
+ */
+module Models {
+    /**
+     * Log package represents the logs collected by each Cockroach node.
+     */
+    export module Log {
+        import promise = _mithril.MithrilPromise;
+
+        export interface LogResponseSet {
+            d: Proto.LogEntry[]
+        }
+
+        // TODO(bram): Move these into css classes.
+        var tableStyle = "border-collapse:collapse; border - spacing:0; border - color:#ccc";
+        var thStyle = "font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:#ccc;color:#333;background-color:#efefef;text-align:center";
+        var tdStyleOddFirst = "font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:#ccc;color:#333;background-color:#efefef;text-align:center";
+        var tdStyleOdd = "font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:#ccc;color:#333;background-color:#f9f9f9;text-align:center";
+        var tdStyleEvenFirst = "font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:#ccc;color:#333;background-color:#efefef;text-align:center";
+        var tdStyleEven = "font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:#ccc;color:#333;background-color:#fff;text-align:center";
+
+        export class Entries {
+
+            startTime = Utils.chainProp(this, <number>null);
+            endTime = Utils.chainProp(this, <number>null);
+            max = Utils.chainProp(this, <number>null);
+            level = Utils.chainProp(this, <string>null);
+
+            private _url():string {
+                var url = "/_status/local/log";
+                if (this.level() != null) {
+                    url += "/" + this.level();
+                }
+                url += "?";
+                if (this.startTime() != null) {
+                    url += "startTime=" + this.startTime().toString() + "&";
+                }
+                if (this.endTime() != null) {
+                    url += "entTime=" + this.endTime().toString() + "&";
+                }
+                if (this.max() != null) {
+                    url += "max=" + this.max().toString() + "&";
+                }
+                return url;
+            }
+
+            private _data = new Utils.QueryCache((): promise<Proto.LogEntry[]> => {
+                return m.request({ url: this._url(), method: "GET", extract: nonJsonErrors })
```

Comment:
This has been factored into the Utils.Http package.  `Utils.Http.Get(this._url)` will work.